intro.Rmd

```diff
@@ -2,19 +2,12 @@
 
 ## Folder structure 
 
-In clinical trial development, developers need to organize source code to deliver 
-Study Data Tabulation Model (SDTM) or Analysis Dataset Model (ADaM) datasets 
-and tables, listings, and figures (TLFs).
-For example, a Phase 3 clinical trial typically consists hundreds of TLFs. 
-A consistent and well-defined folder structure is crucial 
-to managing a clinical trial analysis and reporting (A&R) project. 
-
-We recommend using R package folder structure to organize all the A&R related 
-source code and documentation for a clinical trial A&R project. 
-By using the R package folder structure, 
-you can follow the convention to organize your code as other contributors on CRAN.
-This consistent approach simplifies the communication for all developers 
-within and across organizations. 
+In clinical trial development, source code needs to be developed and maintained to generate and deliver 
+Study Data Tabulation Model (SDTM), Analysis Dataset Model (ADaM) datasets and tables, listings, and figures (TLFs).
+A typical example is a Phase 3 clinical trial where hundreds of TLFs are required for submission. 
+Considering the amount of programs required for such an effort, a consistent and well-defined folder structure is crucial in managing a clinical trial analysis and reporting (A&R) project. 
+
+During the course of this book we will provide a recommended folder structure that is aligned with CRAN submissions of R packages.
```

Comment:
As the book can be a standalone document, I try to avoid mention 'the course" 

```suggestion
We recommend using 
[R package folder structure](https://github.com/rstudio/cheatsheets/raw/master/package-development.pdf) 
to organize all the A&R related source code and documentation for a clinical trial A&R project. 
The R package folder structure is well defined and widely used in R community through repositories (e.g. CRAN).

The consistent approach of using R package folder structure 
simplifies the communication for all developers within and across organizations.
```